HousingRepairsSchedulingApi/Services/Drs/DrsService.cs

```diff
@@ -140,7 +140,26 @@ public async Task<int> CreateOrder(string bookingReference, string sorCode, stri
             {
                 var createOrderResponse = await _drsSoapClient.createOrderAsync(new createOrder(createOrder));
 
-                LambdaLogger.Log($"Successfully called createOrderAsync with {bookingReference}. createOrderResponse: {JsonSerializer.Serialize(createOrderResponse)}");
+                LambdaLogger.Log($"Successfully called createOrderAsync with {bookingReference}. createOrderResponse: {createOrderResponse}");
+
+                if (createOrderResponse == null)
+                {
+                    LambdaLogger.Log($"'createOrderResponse' for booking reference {bookingReference} is null");
+                }
+                else
+                {
+                    LambdaLogger.Log($"'createOrderResponse' for booking reference {bookingReference} is not null");
+
+                }

```

Comment:
This could be simplified to a single line

```suggestion
        
LambdaLogger.Log($"'createOrderResponse' for booking reference {bookingReference} is {createOrderResponse == null ? "null" : "not null"}");

```

But keep this if you think its more readable. Also, same for below.